test/ci-operator-integration/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+function sort_nested_JSON_to_file {
+   jq --argfile a $1 -n 'def post_recurse(f): def r: (f | select(. != null) | r), .; r; def post_recurse: post_recurse(.[]?); ($a | (post_recurse | arrays) |= sort) as $a | $a' > $2
```

Comment:
Other than the complexity of this, we don't want to reorder all fields. Some of them (e.g. the `command` field in a container) need to be in a particular order.

We should identify which specific objects needs to be ordered.

Reply:
Ordering can also be implemented in the logger, in go, if it gets anywhere near this level of complexity.

Reply:
We should order explicitly in the Go code where it's needed

Reply:
The drylogger holds a `[]runtime.Object` that needs to be sorted in some way. I will try to check it.